libs/s25main/mapGenerator/objects/Harbors.cpp

```diff
@@ -36,8 +36,9 @@ std::vector<double> Harbors::NextHarborDistance(const Coastline& coastline,
                                                 const std::vector<Position>& harbors,
                                                 const MapExtent& size)
 {
-    double currentDistance = double(size.x + size.y);
-    int coastNodes = coastline.size();
+    auto currentDistance = double(size.x + size.y);
```

Comment:
Can you avoid C-Style casts (`(type)(expr)` or `type(expr)`) please? Use `static_cast<double>(...)` etc. The current code might do more than one wants (don't ask...) and I'd like to completely disallow C-Style casts in the code base sometime in the future.

Reply:
**Fixed**
I replaced all C-Style casts by using `static_cast` instead. Hopefully I didn't miss any.